config/local.json

```diff
@@ -1,4 +1,10 @@
 {
+    "database": {
```

Comment:
For local settings we should use ENV variables, because its easier to change these settings in .env file when needed than pushing unnecessary code changes to the repository. 

You can check env variables in each `*.config.ts` file - for example you can configure database using variables:
```
POSTGRES_HOST=gs-store-web-db
POSTGRES_NAME=gs-store
POSTGRES_USER=gs-store-db-user
POSTGRES_PASSWORD=abcd1234
```

Reply:
I was using these values, but they didn't get propagated through the app. Not sure what it was at that time.

I'll double check now, using the docker compose to set the env vars.

Reply:
Indeed, it was just needed to use the env vars and they were propagated through the app.